### PR TITLE
Upgrade Fluent bit 0.11.9

### PR DESCRIPTION
**Pull request by edsiper**
This patch do the following changes:
    
- Use the new Fluent Bit v0.11.9
- Bump chart version to AppVersion 0.1.1
- Update description keywords
    
For more details about Fluent Bit v0.11.9 changes, please visit the following release notes:
    
http://fluentbit.io/announcements/v0.11.9/

### CODE REVIEW

**Comment by kfox1111** [comment]
This looks good to me.

**Comment by kfox1111** [comment]
LGTM

File: stable/fluent-bit/Chart.yaml [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
 icon: http://fluentbit.io/assets/img/logo1-default.png
 home: http://fluentbit.io
 maintainers:
-- name: edsiper
-  email: eduardo@treasure-data.com
-- name: kfox1111
-  email: Kevin.Fox@pnnl.gov
+  - name: kfox1111
+    email: Kevin.Fox@pnnl.gov
+  - name: Eduardo Silva
```
Comment (prydonius) [inline stable/fluent-bit/Chart.yaml]: Any reason you added yourself as a maintainer twice? Also can you use `edsiper` for the name.

File: stable/fluent-bit/Chart.yaml [hunk 2]
```diff
@@ -1,16 +1,20 @@
 name: fluent-bit
-version: 0.1.0
-appVersion: 0.11.8
+version: 0.1.2
```
Comment (prydonius) [inline stable/fluent-bit/Chart.yaml]: should this be 0.1.1?

**Review by prydonius — CHANGES_REQUESTED** [review]

File: stable/fluent-bit/Chart.yaml (hunk 2 above)
Comment (edsiper) [reply stable/fluent-bit/Chart.yaml]: fixed.

File: stable/fluent-bit/Chart.yaml (hunk 1 above)
Comment (edsiper) [reply stable/fluent-bit/Chart.yaml]: thanks, using github name now.

**Comment by edsiper** [comment]
@prydonius thanks for the review, I've updated the commits with the changes suggested

**Comment by prydonius** [comment]
thanks @edsiper!

@k8s-bot ok to test

**Review by prydonius — APPROVED** [review]

**Pull request merged**